### ISSUE TITLE
Two-way sync?

### ISSUE THREAD

**Issue by cbrnr** [quarto-dev/quarto-vscode#31]
VS Code supports two-way sync for Markdown documents. That is, if you scroll in the source the preview will sync to the corresponding locations and vice versa. This would be a really useful feature for the Quarto Preview too – is this planned by any chance?

**Comment by jjallaire** [comment]
Yes, this is something we'd definitely like to implement!

**Comment by marcel-goldschen-ohm** [comment]
Yes, this would be enormously helpful. Also, when rendering to a revealjs slideshow it would be great if the preview jumped to the slide containing the currently selected element in the code file.
